### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by calonmerc** [nebhead/PiFire#125] [General]
Would love if there was a community Discord where everyone could share their latest smokes with PiFire, PID tunings they've done, builds, chat about ideas, etc. The discussions on GH are probably 50% of what I'd love to see, but just throwing it out there!

**Comment by nebhead** [comment]
Yeah, that would be great!  I mean, I'm a newb when it comes to Discord so any guidance on how to set something like that up would be cool.

**Reply by calonmerc** [reply]
Generally Discord is easy to set up, it's also free. If you want, I can look through some of the servers I'm in for other open source projects that I think have set up their channels, rules, etc very well and send that!

**Reply by nebhead** [reply]
> Generally Discord is easy to set up, it's also free. 

Free is great!  I'll poke around and see what I can do to get something started.  Looks like fun!

> If you want, I can look through some of the servers I'm in for other open source projects that I think have set up their channels, rules, etc very well and send that!

Absolutely!  Thank you so much!

**Comment by calonmerc** [comment]
Here are a couple different channel layouts that seem to work well _ish_
![image](https://user-images.githubusercontent.com/6835832/193873698-79d20409-1325-4366-9636-9e1f2e4c32f1.png)
![image](https://user-images.githubusercontent.com/6835832/193873905-c1d39d1b-8b7b-4d85-aca2-a1045b1c4140.png)
![image](https://user-images.githubusercontent.com/6835832/193874167-36cd74b2-e890-47ae-9621-0160cddf830e.png)

Finally, I include this one, as it's very basic, but the only other hardware project I'm involved with. 
![image](https://user-images.githubusercontent.com/6835832/193874432-5199a6cd-ebe1-4df0-9393-eb889620b7fa.png)

**Reply by nebhead** [reply]
Thanks!  Good ideas there.  I will take a swing at it.

**Reply by nebhead** [reply]
OK, got a veeeerrry basic server setup here:  https://discord.gg/F9mbCrbrZS  

I welcome all to join as they wish. :)  We will see how this all goes.

**Reply by calonmerc** [reply]
May want to add that info to the readme, wherever you're ready! I've joined!
